### ISSUE TITLE
Remove (old) android-device and re-add it

### ISSUE THREAD

**Issue by wschoot** [keybase/client#15801]
Hi,

I've been using keybase on android in the past and that device is currently linked to my keybase account. Now, I wanna re-add the same device (after I had to factory reset it) and it only allows me to do so after verifying with the freshly installed device, which isn't gonna work.

I have my private key saved on keybase.io, I can still sign proof for new sites etc. I can also extract the private key (I made an offline copy of it). So, I *think* most of the needed bits are still there. However, I cannot login on CLI or on my android as it has one option only; the lost installation of the android application.

I can't revoke the device, I can't add a device (on the site), nor can't I create paper proof or some other way to get back on track.

I prefer not to reset my account as I think this means redoing all proofs and reconnecting to everyone.

TL:DR How can I re-add my reinstalled device if it's the only device I have, but still have access to the private key on my keybase.io account?

Thanks!
